gen-ai/sample-claude-text.json

```diff
@@ -0,0 +1,111 @@
+{
+  "variable": [
+    {
+      "value": "https://api.anthropic.com/v1/messages",
+      "key": "api.url"
+    },
+    {
+      "value": "<your-prompt-here>",
+      "current": "",
+      "key": "textInput"
+    },
+    {
+      "value": "claude-3-opus-20240229",
+      "key": "llm"
+    },
+    {
+      "value": "1",
+      "current": "",
+      "key": "temp"
+    },
```

Comment:
I think it would be a good enhancement (maybe next rev) to add top_p and top_k  https://docs.anthropic.com/en/api/messages

Reply:
Do you think there is a risk that adding `top_p` and `top_k` as variables would confuse users who are less familiar with those concepts?